data/base-contacts.yaml

```diff
@@ -0,0 +1,3356 @@
+contacts:
```

Comment:
Sorry, when I say profiles folder I meant /lich/scripts/profiles/ which is the same place the yourname-setup.yaml file is.